cfme/fixtures/ansible_fixtures.py

```diff
@@ -21,7 +22,7 @@ def ansible_repository(appliance, wait_for_ansible):
     repositories = appliance.collections.ansible_repositories
     repository = repositories.create(
         name=fauxfactory.gen_alpha(),
-        url="https://github.com/quarckster/ansible_playbooks",
+        url=cfme_data.ansible_links.playbook_repositories.embedded_ansible,
```

Comment:
Now that this is coming from a yaml lookup, it needs to be wrapped in a try block to skip the test when this is missing, instead of raising KeyError/AttributeError.